website/components/TableSearch.tsx

```diff
@@ -0,0 +1,74 @@
+import "instantsearch.css/themes/reset.css";
+import React from "react";
+import Link from "next/link";
+import algoliasearch from "algoliasearch/lite";
+import {
+  InstantSearch,
+  connectSearchBox,
+  connectStateResults,
+} from "react-instantsearch-dom";
+
+const searchClient = algoliasearch(
+  "0OUG5EUZ6H",
```

Comment:
I created this Algolia App on a personal account. We can create a CloudQuery one.
This is a client side API key, so safe to put in the code